### PR TITLE
Created Contact us page

### PR DESCRIPTION
**Pull request by vadlachary2002**
1.Some changes in navbar has been done
2.Created Contact us page
image
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,11 +4,11 @@
         <meta charset="utf-8">
         <meta content="IE=edge" http-equiv="X-UA-Compatible" />
         <meta name="viewport" content="width=device-width, user-scalable=no" />
-        <title><%= htmlWebpackPlugin.options.title %></title>
-        <link rel="icon" href="public/favicon.ico">
+        <title>shorebirdie.com</title>
+        <link rel="icon" href="public/logo.png">
```
Comment (maharjunm) [inline public/index.html:8]: you can remove this link

File: src/pages/ContactUs/ContactUs.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+:root {
+  --blue:#0064a9;
+  --yellow: #ffb12a;
+	--lightblack:#78745c;
```
Comment (maharjunm) [inline src/pages/ContactUs/ContactUs.scss]: indentation not correct.

File: src/components/NavBar/NavBar.scss [hunk 3]
```diff
@@ -1,17 +1,29 @@
+:root {
+  --blue:#0064a9;
+  --yellow: #ffb12a;
+	--lightblack:#78745c;
```
Comment (maharjunm) [inline src/components/NavBar/NavBar.scss]: indentation is wrong

File: public/index.html (hunk 1 above)
Comment (vadlachary2002) [reply public/index.html:8]: shorebirdie logo teseyala annaya title bar lo nundi

**Review by maharjunm — APPROVED** [review]

**Pull request merged**